ezcater_rubocop.gemspec

```diff
@@ -44,15 +44,15 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = ">= 2.6"
 
-  spec.add_development_dependency "bundler", "~> 2.1"
+  spec.add_development_dependency "bundler", "~> 2.3"
   spec.add_development_dependency "pry-byebug"
-  spec.add_development_dependency "rake", "~> 12.3"
-  spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency "rake", "~> 13.0"
+  spec.add_development_dependency "rspec", "~> 3.11"
   spec.add_development_dependency "rspec_junit_formatter"
-  spec.add_development_dependency "simplecov", "< 0.18.0"
+  spec.add_development_dependency "simplecov"
 
-  spec.add_runtime_dependency "parser", "!= 2.5.1.1"
-  spec.add_runtime_dependency "rubocop", "~> 1.16.0"
-  spec.add_runtime_dependency "rubocop-rails", "~> 2.10.1"
-  spec.add_runtime_dependency "rubocop-rspec", "~> 2.3.0"
+  spec.add_runtime_dependency "parser", "~> 3.1"
+  spec.add_runtime_dependency "rubocop", "~> 1.26"
+  spec.add_runtime_dependency "rubocop-rails", "~> 2.14"
+  spec.add_runtime_dependency "rubocop-rspec", "~> 2.9"
```

Comment:
Thoughts on incrementally adopting these changes in 3.x so that apps dependent on ezcater_rubocop can benefit from them?

Reply:
Can you re-review it with my new dependency specifications?